gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceConfiguration.java

```diff
@@ -70,6 +70,9 @@ public class GobblinServiceConfiguration {
   @Getter
   private final boolean isHelixManagerEnabled;
 
+  @Getter
+  private final boolean isSpecStoreChangeMonitorEnabled;
```

Comment:
I was thinking that as well because we do not one turned on without the other. However neither the `warmStandbyEnabled` or `specStoreChangeMonitorEnabled` terms encapsulate all the multi-leader mode changes as a whole. I can create a configuration key `multiLeaderModeEnabled` which we can use in both places. What do you think?

Reply:
Multi leaders is more about partitioning that each host can handle part of the work. All the features we are doing here are for warm standby so that when one leader is down, other hosts can take over the job without gap. That's why I use the warm standby as the config name, let me know if you have other thoughts.

Reply:
I see that is a reasonable name then. We will use this to keep all or any hosts up to date with leader who is accepting all the API requests in warm standby mode. Only leader node will be able to execute. Changing to use `warmStandbyEnabled` key.